### ISSUE TITLE
Issue Start-HardenADLocalGroups

### ISSUE THREAD

**Issue by msreport** [HardenAD/SecureAD#40]
There is a mistake in the log path in Start-HardenADLocalGroups

if (!(Test-Path "$env:windir\Harden\Logs")) {
        try {
            New-Item -Path "$env:windir\Harden\Logs" -ItemType Directory -Force

**Comment by msreport** [comment]
Line 64 and 65

**Comment by kdvhh** [comment]
```
$ErrorActionPreference = "SilentlyContinue"
$LogPath = "$env:windir\Harden\Logs"

    if (!(Test-Path $LogPath)) {
        try {
            New-Item -Path $LogPath -ItemType Directory -Force
        }
        catch {
            Write-Error -ErrorRecord $Error[0] `
                -RecommendedAction "Unable to create the Harden folder. Make sure you can write to this location: {0}." -f "$env:windir\Logs\"
            exit
        }
    }
    
    [LoggerFactory]::Initialize($LogPath)
```

Ok comme ça ? @msreport

**Comment by LoicVeirman** [comment]
fixed in 2.9.7.
